internal/dicts/wordmap.go

```diff
@@ -32,9 +32,17 @@ func (m *WordMap) AddFrom(other WordMap) {
 }
 
 // Add a single mapping from one word to another to the WordMap.
+//
+// This function panics if an empty string is provided as first argument.
```

Comment:
This should be update to include that it `panic`s if the second argument is an empty string.

```suggestion
// This function panics if an empty string is provided as first or second
// argument.
```